spec/catalog.yaml

```diff
@@ -150,6 +169,18 @@ components:
           example: { "created_at": "1452120468" }
           default: { }
 
+    ListNamespacesResponse:
```

Comment:
should we add pagination?

Reply:
With the same answer, let's do it later? To simplify the api lands. And make the pagination to be an improvement.

Reply:
Sounds good!